src/style.css

```diff
@@ -0,0 +1,72 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't reset margins and paddings using * selector. Better specify the exact tags you want to reset

Reply:
not fixed
![image](https://user-images.githubusercontent.com/122989585/232333859-bf263fc1-2eab-4f61-a4b4-78428d191dfc.png)